### ISSUE TITLE
Add to CocoaPods?

### ISSUE THREAD

**Issue by cstigler** [mysteriouspants/ArgumentParser#12]
Hi @mysteriouspants , would you be open to adding this library to CocoaPods so it could be more easily included in other projects?

I see in #6 that you removed the podspec because CoreParse removed themselves from CocoaPods, which makes sense. However, since then I also see that you've removed CoreParse as a dependency entirely. So at this point, is there any blocker to putting this up on CocoaPods?

**Comment by mysteriouspants** [comment]
Yeah, I have no problems with it. It looks like CocoaPods has evolved since it was just a bunch of PodSpec files in a giant Git repo!

I'll try to put some time into that this weekend.

**Comment by cstigler** [comment]
Thank you! Let me know if I can help at all. I just added this as a dependency to [SelfControl](http://selfcontrolapp.com/) - currently vendor'd in but it'll be great to have it as a pod for easier maintenance and cleanliness.
